websites/S/Sonic Retro/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "BigBoi",
+		"id": "853932304257908746"
+	},
+	"service": "Sonic Retro",
+	"description": {
+		"en": "'Second only to Sega' - Sonic Retro is the Sonic scene's GFDL input and output wiki resource for collecting data about Sonic the Hedgehog that is open to the entire community."
+	},
+	"url": "sonicretro.org",
+	"regExp": "([a-z0-9-]+[.])*(sonicretro[.]org)[/]",
```

Comment:
you only support info.* and forums.* so you should add these to the URL array instead of making a regexp